### PR TITLE
Add example main page

### PR DESCRIPTION
**Pull request by d0rich**

### CODE REVIEW

File: content/4.examples/0.index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    + sendPayment()
+    + registerParticipant()
+    + getPayment()
+    + queryPayments()
+  }
+```
+
+## Template
+
+In order to have access to Corda flows and services, `GevamuFacade` class should be implemented as Corda service.
```
Comment (lingvolab) [inline content/4.examples/0.index.md:59]:
```suggestion
In order to have access to Corda flows and services, the `GevamuFacade` class should be implemented as a Corda service.
```

**Review by lingvolab — CHANGES_REQUESTED** [review]

File: content/4.examples/0.index.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+repositories {
+    mavenCentral()
+    // Repository with corda artifacts
+    maven("https://software.r3.com/artifactory/corda")
+}
+
+dependencies {
+    // Core corda dependencies
+    implementation("net.corda:corda-core:4.9.3")
+    // Download SDK https://gevamu.github.io/corda-payments-sdk/
```
Comment (ASitnikov) [inline content/4.examples/0.index.md:38]:
```
// Gevamu dependencies
```
We assume that they will be published on central repository. There is no need to mention download.

File: content/4.examples/0.index.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+```kotlin [build.gradle.kts]
+repositories {
+    mavenCentral()
+    // Repository with corda artifacts
+    maven("https://software.r3.com/artifactory/corda")
+}
+
+dependencies {
+    // Core corda dependencies
```
Comment (ASitnikov) [inline content/4.examples/0.index.md:36]:
```suggestion
    // Core Corda dependencies
```
I'm not sure if R3 writes Corda or corda. If it's corda, please disregard this comment.

**Review by ASitnikov — CHANGES_REQUESTED** [review]

File: content/4.examples/0.index.md (hunk 3 above)
Comment (d0rich) [reply content/4.examples/0.index.md:36]: You are right, we should write "Corda"

File: content/4.examples/0.index.md (hunk 2 above)
Comment (d0rich) [reply content/4.examples/0.index.md:38]: Looks like this comment is misleading.

```suggestion
    // Gevamu dependencies
```

**Review by ASitnikov — APPROVED** [review]
Looks good to me

**Pull request merged**